lib/internal/source_map/source_map_cache.js

```diff
@@ -281,56 +307,44 @@ function sourcesToAbsolute(baseURL, data) {
   return data;
 }
 
-// WARNING: The `sourceMapCacheToObject` and `appendCJSCache` run during
-// shutdown. In particular, they also run when Workers are terminated, making
-// it important that they do not call out to any user-provided code, including
-// built-in prototypes that might have been tampered with.
+// WARNING: The `sourceMapCacheToObject` run during shutdown. In particular,
+// they also run when Workers are terminated, making it important that they do
+// not call out to any user-provided code, including built-in prototypes that
+// might have been tampered with.
```

Comment:
Only one function to run now:

```suggestion
// WARNING: `sourceMapCacheToObject()` is run during shutdown. In particular,
// it's also run when Workers are terminated, making it important that it does
// not call out to any user-provided code, including built-in prototypes that
// might have been tampered with.
```